### ISSUE TITLE
roof.egybest.org always detect adblock

### ISSUE THREAD

**Issue by EslamGamalEssa** [uBlockOrigin/uAssets#8319]
### URL(s) where the issue occurs

`https://roof.egybest.org`

### Describe the issue

this site always detect adblock and it have more than 100 ads in minute

### Screenshot(s)

![ADBLOCK DETECTED](https://user-images.githubusercontent.com/22161118/101155330-9d93bd00-362f-11eb-8c1b-f402eb1f626e.jpg)

### Versions

- Browser/version: google chrome / 87.0.4280.88 (Official Build) (64-bit)
- uBlock Origin version: 1.31.2

### Settings

default settings

### Notes

it all depend on 3 js files

```
https://cdn-static.egybest.net/static/global/js/adblock.js

https://cdn-static.egybest.net/static/global/js/jquery-3.3.1.min.js

https://cdn-static.egybest.net/packed/G78uOQIvP42dDsB.js
```

when i block the first two in my filters won't detect the ad block but buttons in page won't work

**Comment by krystian3w** [comment]
AdGuard also changed method:

https://github.com/AdguardTeam/AdguardFilters/commit/32e155c7e9fc0b4e0f98c714c9c3940085f77167 
https://github.com/AdguardTeam/AdguardFilters/commit/19d5729c361ab8455df877afbcabffdc1718764a

@lain566

**Comment by EslamGamalEssa** [comment]
> AdGuard also changed method:
> 
> [AdguardTeam/AdguardFilters@32e155c](https://github.com/AdguardTeam/AdguardFilters/commit/32e155c7e9fc0b4e0f98c714c9c3940085f77167)
> [AdguardTeam/AdguardFilters@19d5729](https://github.com/AdguardTeam/AdguardFilters/commit/19d5729c361ab8455df877afbcabffdc1718764a)
> 
> @lain566 can you test:
> 
> `/^\/(?!api\?call=)[a-zA-Z0-9]{5,20}$/`?

sorry can you explain in steps ?

**Comment by NonVolatileHuman** [comment]
*Other mirror links to the same website

egy.best
egybest.asia
egybest.bid
egybest.biz
egybest.cheap
egybest.co
egybest.com
egybest.cool
egybest.cz
egybest.life
egybest.ltd
egybest.ist
egybest.name
egybest.net
egybest.network
egybest.ninja
egybest.nl
egybest.online
egybest.org
egybest.pw
egybest.rocks
egybest.site
egybest.xyz
egybest.zone

tries multiple filter edits but didn't work either (like this):
/(\/install\.|\/track\.|\/jump\/|\&clickid=|\&subid=|\&campaignid=|\?pid=|\?zoneid=)/$popup,domain=egy.best|egybest.asia|egybest.bid|egybest.biz|egybest.cheap|egybest.co|egybest.com|egybest.cool|egybest.cz|egybest.life|egybest.ltd|egybest.ist|egybest.name|egybest.net|egybest.network|egybest.ninja|egybest.nl|egybest.online|egybest.org|egybest.pw|egybest.rocks|egybest.site|egybest.xyz|egybest.zone
@@$generichide,domain=egy.best|egybest.asia|egybest.bid|egybest.biz|egybest.cheap|egybest.co|egybest.com|egybest.cool|egybest.cz|egybest.life|egybest.ltd|egybest.ist|egybest.name|egybest.net|egybest.network|egybest.ninja|egybest.nl|egybest.online|egybest.org|egybest.pw|egybest.rocks|egybest.site|egybest.xyz|egybest.zone
@@/click.php$popup,~third-party,domain=egy.best|egybest.asia|egybest.bid|egybest.biz|egybest.cheap|egybest.co|egybest.com|egybest.cool|egybest.cz|egybest.life|egybest.ltd|egybest.ist|egybest.name|egybest.net|egybest.network|egybest.ninja|egybest.nl|egybest.online|egybest.org|egybest.pw|egybest.rocks|egybest.site|egybest.xyz|egybest.zone
@@||cdn-static.egybest.net/static/global/js/adblock.js$script,domain=egy.best|egybest.asia|egybest.bid|egybest.biz|egybest.cheap|egybest.co|egybest.com|egybest.cool|egybest.cz|egybest.life|egybest.ltd|egybest.ist|egybest.name|egybest.net|egybest.network|egybest.ninja|egybest.nl|egybest.online|egybest.org|egybest.pw|egybest.rocks|egybest.site|egybest.xyz|egybest.zone
@@||stremanp.com/afu.php?zoneid=$popup,domain=egy.best|egybest.asia|egybest.bid|egybest.biz|egybest.cheap|egybest.co|egybest.com|egybest.cool|egybest.cz|egybest.life|egybest.ltd|egybest.ist|egybest.name|egybest.net|egybest.network|egybest.ninja|egybest.nl|egybest.online|egybest.org|egybest.pw|egybest.rocks|egybest.site|egybest.xyz|egybest.zone

**Comment by lain566** [comment]
> 
> @lain566 can you test:
> 
> `/^\/(?!api\?call=)[a-zA-Z0-9]{5,20}$/`?`

@krystian3w It's already added
https://github.com/uBlockOrigin/uAssets/blob/e6b32e513d131c3e2bb5981ddc8aa46b786ce279/filters/filters.txt#L16932

**Comment by krystian3w** [comment]
Chromium no reject use `<object>` / `<embed>` / `<iframe>`?

**Comment by Mennaruuk** [comment]
So most of the mirrors above don't work anymore, and the one that worked for me didn't detect adblock. Should this issue be closed?

**Comment by lain566** [comment]
By clicking on the download button, can you download the movie? or does the same page always open?

**Comment by Mennaruuk** [comment]
Correct. The same page opens up.
![imageedit_2_8934362341](https://user-images.githubusercontent.com/52135169/108746730-b616c380-750a-11eb-9abb-eeb45503850e.png)

That's because of the site itself (the only working site from the mirror list above is `egybest.nl`). It seems shady. This is what I got when I pasted the Download link of one movie into Redirect Detective:

![imageedit_1_9458444766](https://user-images.githubusercontent.com/52135169/108746681-a6977a80-750a-11eb-9ed4-f8be3c57ef08.png)
